### DISCUSSION TITLE
What do the colors `copy_mode_active_highlight_*` represent?

### DISCUSSION THREAD

**Discussion by MrPandir** [wezterm/wezterm#5874] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I just started learning `wezterm` and configuring it, and I can't figure out what colors settings like `copy_mode_active_highlight_*` affect?

Here is the configuration I am experimenting with:
```lua
local wezterm = require("wezterm")
local config = wezterm.config_builder()

config.colors = {
	foreground = "white",
	background = "black",

	cursor_bg = "#80baa1",
	cursor_fg = "black",
	cursor_border = "none",

	selection_fg = "none",
	selection_bg = "#234a4c",

	scrollbar_thumb = "#222222",

	split = "#444444",

	-- ansi = {
	-- 	"black",
	-- 	"maroon",
	-- 	"green",
	-- 	"olive",
	-- 	"navy",
	-- 	"purple",
	-- 	"teal",
	-- 	"silver",
	-- },
	-- brights = {
	-- 	"grey",
	-- 	"red",
	-- 	"lime",
	-- 	"yellow",
	-- 	"blue",
	-- 	"fuchsia",
	-- 	"aqua",
	-- 	"white",
	-- },

	-- Arbitrary colors of the palette in the range from 16 to 255
	-- indexed = { [136] = "#af8700" },

	-- When the IME, a dead key or a leader key are being processed and are effectively
	-- holding input pending the result of input composition, change the cursor
	-- to this color to give a visual cue about the compose state.
	compose_cursor = "orange",

	-- Colors for copy_mode and quick_select
	-- In copy_mode, the color of the active text is:
	-- 1. copy_mode_active_highlight_* if additional text was selected using the mouse
	-- 2. selection_* otherwise
	copy_mode_active_highlight_bg = { Color = "#000000" },
	-- use `AnsiColor` to specify one of the ansi color palette values
	-- (index 0-15) using one of the names "Black", "Maroon", "Green",
	--  "Olive", "Navy", "Purple", "Teal", "Silver", "Grey", "Red", "Lime",
	-- "Yellow", "Blue", "Fuchsia", "Aqua" or "White".
	copy_mode_active_highlight_fg = { AnsiColor = "Black" },
	copy_mode_inactive_highlight_bg = { Color = "#52ad70" },
	copy_mode_inactive_highlight_fg = { AnsiColor = "White" },

	quick_select_label_bg = { Color = "peru" },
	quick_select_label_fg = { Color = "#ffffff" },
	quick_select_match_bg = { AnsiColor = "Navy" },
	quick_select_match_fg = { Color = "#ffffff" },
}

return config
```

As far as I understand, this should somehow change the colors of the selection in copy mode. But in my case, I don't see any changes, and the selection corresponds to the color `selection_*`.

<img width="817" src="https://github.com/user-attachments/assets/181f2d0a-3ecd-41f7-8418-d99b329eb095">

In the screenshot you can see that I am in copy mode, and the highlight color is `selection_*`, when I expect it to be `copy_mode_active_highlight_*`.

I couldn't find any information or demonstration of this in the documentation or discussion. I’ll also let you know that I tried to comment out the `selection_*` lines and got the same result, except that the color changed to the default one.

I need someone to demonstrate how this works or describe the actions I need to take to see these colors.
I also don't understand the difference between `active` and `inactive`.
I would be very grateful for your help.

**Comment by bgracie** [comment]
I have the same issue.  Related to https://github.com/wezterm/wezterm/discussions/7087 I believe.
